src/qt/walletframe.h

```diff
@@ -30,7 +30,7 @@ class WalletFrame : public QFrame
     void setClientModel(ClientModel *clientModel);
 
     bool addWallet(const QString& name, WalletModel *walletModel);
-    bool setCurrentWallet(const QString& name);
+    WalletModel* setCurrentWallet(const QString& name);
```

Comment:
nit: From the name this sounds like a setter, but it returns a WalletModel. Maybe add documentation why, or change the function name?